### DISCUSSION TITLE
Display multiple patches on one web page?

### DISCUSSION THREAD

**Discussion by ZauggT** [cables-gl/cables_docs#422] [Answers & Questions] [answered]
I am quite inexperienced with Cables.gl and building websites. But I am trying to build some kind of gallery in the form of a website for a project at my university. I would like to display the different patches I have made on one web page at the same time.
Is there a way to display multiple patches on one web page?

**Comment by steam0r** [comment, marked as answer]
i think the easies way would be to include each of them in an iframe, that way you can use the index.html included in the export. otherwise it should also not be a problem to instanciate multiple patches on one site as described here: 

https://cables.gl/docs/4_export_embed/embedding/embedding

for some more examples, check:

https://github.com/cables-gl/cables-embedding

**Reply by ZauggT** [reply]
Yes that makes it much easier, thank you!
